tests/openvino/test_modeling.py

```diff
@@ -1899,10 +1913,16 @@ def test_compare_to_transformers(self, model_arch):
     @parameterized.expand(SUPPORTED_ARCHITECTURES)
     def test_generate_utils(self, model_arch):
         model_id = MODEL_NAMES[model_arch]
-        model = OVModelForVisualCausalLM.from_pretrained(model_id, export=True)
-        preprocessor = get_preprocessor(model_id)
-        question = "<image>\nDescribe image"
-        inputs = preprocessor(images=self.IMAGE, text=question, return_tensors="pt")
+        model = OVModelForVisualCausalLM.from_pretrained(
+            model_id, export=True, trust_remote_code=model_arch in self.REMOTE_CODE_MODELS
+        )
```

Comment:
no need for the export=True
```suggestion
        model = OVModelForVisualCausalLM.from_pretrained(
            model_id, trust_remote_code=model_arch in self.REMOTE_CODE_MODELS
        )
```